wasm/src/account/private_key.rs

```diff
@@ -114,6 +114,12 @@ impl From<PrivateKeyNative> for PrivateKey {
     }
 }
 
+impl Into<PrivateKeyNative> for PrivateKey {
+    fn into(self) -> PrivateKeyNative {
+        self.0
+    }
+}
+
```

Comment:
The `From` trait is always preferred in Rust over `Into`. It will still allow you to use the `into()` function.

```suggestion
impl From<PrivateKey> for PrivateKeyNative {
    fn from(private_key: &PrivateKey) -> Self {
        private_key.0
    }
}

impl From<&PrivateKey> for PrivateKeyNative {
    fn from(private_key: &PrivateKey) -> Self {
        *private_key.0
    }
}
```

Reply:
That makes a ton of sense, thanks. I saw that in the Rust docs, but I was hesitant to change the underlying native private key in snarkVM -- adding the trait in the wasm code makes a ton of sense!